README.md

```diff
@@ -30,6 +30,14 @@ The release notes can be found [here](https://github.com/dnvriend/akka-persisten
 
 For change log prior to v3.2.0, visit [Version History Page (wiki)](https://github.com/dnvriend/akka-persistence-jdbc/wiki/Version-History).
 
+To migrate from v3.x to v4.x, the database schema needs to be updated. To migrate, see the `<database-name>-migration-v4.sql` files in your corresponding databases [schema directory](https://github.com/dnvriend/akka-persistence-jdbc/tree/master/src/test/resources/schema/). The schema update is both backwards and forwards compatible. If you wish to perform rolling upgrades, canary upgrades, or want to be able to rollback, you will need to enable forwards compatibility, which can be done by setting the following configuration:
+
+```
+akka-persistence-jdbc.migration.v4.forwards-compatibility = true
```

Comment:
I am always confused about this. Shouldn't this be backwards compatibility, since this means the new data is still backwards compatible with akka-persistence-jdbc v3?

Reply:
It depends on both the context and the direction that you're starting from when you're describing the compatibility. In this case, the context is the compatibility between the code and the data, and I'm starting from the direction of the code (since this is a flag that controls the code, it's not a migration being applied to the data). If the new code is compatible with the old data, then that's backwards compatible. If the old code is compatible with the new data, then that's forwards compatible. So this flag ensures that the new code is writing new data that the old code will be forwards compatible with.